COMETwebapp/ViewModels/Components/Shared/ParameterEditors/CompoundParameterTypeEditorViewModel.cs

```diff
@@ -63,6 +83,15 @@ public override async Task OnParameterValueChanged(object value)
             }
         }
 
+        /// <summary>
+        /// Event for when the edit button is clicked
+        /// </summary>
+        public void OnComponentSelected()
+        {
+            CDPMessageBus.Current.SendMessage(new CompoundComponentSelectedEvent(this));

```

Comment:
@antoineatrhea Are we using again the MessageBus inside the ParameterEditors?

Reply:
It's only used to open a popup created inside the ParameterTable. Only the ParameterTale subscribed to it. Easier to do it like that.